index.html

```diff
@@ -36,5 +36,131 @@ <h1 class="heading"> Hey There.<br>I’m Ismail</h1>
             <li><a href="#" alt="meta"><img src='Assets/images/meta.svg' alt="meta"/></a></li>
         </ul>
     </section>
+    <section class="works-section">
+        <h2 class="work-heading">My Recent Works</h2>
+        <section class="post">
```

Comment:
[Optional]

- [ ] It's not mentioned in the requirements but I prefer if the containers wrapping the individual project cards are `div`s instead of `section`s 👍